Sources/TokamakDemo/EnvironmentDemo.swift

```diff
@@ -32,15 +32,31 @@ struct EnvironmentObjectDemo: View {
   }
 }
 
+extension ColorScheme: CustomStringConvertible {
+  public var description: String {
+    switch self {
+    case .dark: return "dark"
+    case .light: return "light"
+        @unknown default:
```

Comment:
<!--
  1 failure:  Case statements s...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakDemo/EnvironmentDemo.swift;
  Line: 40;
-->
- :no_entry_sign: Case statements should vertically align with their enclosing switch statement. (`switch_case_alignment`)